app/controllers/TaskListController.scala

```diff
@@ -50,8 +54,9 @@ class TaskListController @Inject() (
       implicit request =>
         val tasks                = viewModel(request.userAnswers)
         val isSubmitted: Boolean = request.userAnswers.isSubmitted.getOrElse(NotSubmitted).showErrorContent
+        val expiresInDays        = Duration.between(Instant.now(clock), request.userAnswers.createdAt.plus(appConfig.ttlInDays, DAYS)).toDays + 1
```

Comment:
Not a massive deal but similar to how we do with the departure drafts, this logic could live in the backend. Means that we only need to define the mongo ttlInDays in one place. Could we maybe look to utilise that backend logic somehow? cc @MatthewAlexanderBeck thoughts?

Reply:
Yep im fine with that, would make changing the TTL a much simpler task.